### PR TITLE
Fix styles

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* [Design review](https://www.notion.so/xmartlabs/Design-review-a05d0c09e1584261a98cc58d4a410b40?pvs=4)

## Type of change

* [X] Fix
* [ ] Story
* [ ] Chore

## Description of the change

I implemented the requested changes. 

## Screenshot/Execution

https://github.com/xmartlabs/xl-blog/assets/107500715/c3d2e849-e524-4ba0-879b-3b67cd72f039

Captura desde 2023-09-27 16-17-30

## Related PRs
Links to the related PRs

### CODE REVIEW

File: src/components/nav-menu/nav-menu.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const NavMenu = ({ className, openMenu }) => {
           {...commonProps}
           className={classnames(styles.linkTextContainer, className)}
         >
-          <h5 className={classnames(styles.link, {["text__heading__two__separated__blueThree"]: openMenu})}>{commonProps.key}</h5>
+          <h5 className={classnames(styles.link, {["text__heading__two__separated__blueOne"]: openMenu})}>{commonProps.key}</h5>
```
Comment (lauta2912) [inline src/components/nav-menu/nav-menu.jsx:36]:
```suggestion
          <h5 className={classnames(styles.link, { text__heading__two__separated__blueOne: openMenu })}>{commonProps.key}</h5>
```

File: src/pages/404.js [hunk 2]
```diff
@@ -12,8 +12,8 @@ const NotFoundPage = () => {
      <a href="/"><ErrorImage className={styles.img} /></a>
       <div className={styles.subContainer}>
         <title>Not found</title>
-        <h1 className={classnames(styles.heading, "text__heading__one__blueThree")}>You've reached the land of lost URLs.</h1>
-        <p className="text__paragraph__blueThree">
+        <h1 className={classnames(styles.heading, "text__heading__one__blueOne")}>You've reached the land of lost URLs.</h1>
```
Comment (lauta2912) [inline src/pages/404.js:15]:
```suggestion
        <h1 className={classnames(styles.heading, 'text__heading__one__blueOne')}>You've reached the land of lost URLs.</h1>
```

File: src/templates/post.js [hunk 3]
```diff
@@ -163,14 +163,14 @@ const BlogPost = ({ data, children }) => {
         <div className={styles.categoryTagsContainer}>
           <Category data={categoryBlog.displayName} className={styles.category}/>
         </div>
-        <h1 className={classnames(styles.titleContainer, "text__heading__one__blueThree")}>
+        <h1 className={classnames(styles.titleContainer, "text__heading__one__blueOne")}>
```
Comment (lauta2912) [inline src/templates/post.js:166]:
```suggestion
        <h1 className={classnames(styles.titleContainer, 'text__heading__one__blueOne')}>
```

File: src/templates/post.js [hunk 4]
```diff
@@ -163,14 +163,14 @@ const BlogPost = ({ data, children }) => {
...
           { data.mdx.frontmatter.title }
         </h1>
         <div className={styles.authorContainer}>
           <div className={styles.authorInformation}>
             <img src={`/images/${authorBlog.image}`} alt={authorBlog.displayName} className={styles.authorImage} />
             <div className={styles.nameDateTimeContainer}>
-              <Link className={classnames(styles.authorName, "text__paragraph__bold__blueThree")} to={author.profile_url}>{ authorBlog.displayName }</Link>
+              <Link className={classnames(styles.authorName, "text__paragraph__bold__blueOne")} to={author.profile_url}>{ authorBlog.displayName }</Link>
```
Comment (lauta2912) [inline src/templates/post.js:173]:
```suggestion
              <Link className={classnames(styles.authorName, 'text__paragraph__bold__blueOne')} to={author.profile_url}>{ authorBlog.displayName }</Link>
```

**Review by lauta2912 — APPROVED** [review]

File: src/components/nav-menu/nav-menu.jsx (hunk 1 above)
Comment (Melissa-Martinez-XL) [reply src/components/nav-menu/nav-menu.jsx:36]: Done

File: src/pages/404.js (hunk 2 above)
Comment (Melissa-Martinez-XL) [reply src/pages/404.js:15]: Done

File: src/templates/post.js (hunk 3 above)
Comment (Melissa-Martinez-XL) [reply src/templates/post.js:166]: Done

File: src/templates/post.js (hunk 4 above)
Comment (Melissa-Martinez-XL) [reply src/templates/post.js:173]: Done

**Review by diegorosolino — APPROVED** [review]

**Pull request merged**